lib/src/rules/prefer_final_fields.dart

```diff
@@ -119,26 +134,24 @@ class _Visitor extends SimpleAstVisitor<void> {
       return;
     }
 
-    fields.variables.forEach((VariableDeclaration variable) {
-      if (variable == null ||
-          !resolutionMap
-              .elementDeclaredByVariableDeclaration(variable)
-              .isPrivate) {
-        return;
+    var variables = fields.variables;
+    for (int i = 0; i < variables.length; i++) {
```

Comment:
Is this measurably faster than `for (var variable in fields.variables)`?

Reply:
It depends on how we measure it :-)

In order to see the difference I changed `visitFieldDeclaration()` to run the cycle over variables `10000` times. The numbers are:

With `forEach(closure)`: 9500 9497 9450 ms.
With index: 5068 5181 5135 ms.

So, there *is* a difference: `0.44 ms` for approximately the half of the Flutter repo.
But it is very small.

I guess it is not worth changing the style.
I will revert this change.

Reply:
Except that this is one of the ways in Dart to die by a thousand cuts. If we compared using for-each everywhere against using for everywhere, I suspect we'd notice a much bigger difference.